tests/command/update.bats

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bats
```

Comment:
?

Reply:
Not sure anymore why I commented it out, too long ago.

Reply:
Ah maybe because it doesn't work on NC25 but the plan is anyway to drop support for NC25 and to move to NC26 plus.

That would allow us to remove the extra code, which is also in this PR to catch older NC versions.
And we could remove the deprecated time code and replace it like it is advised in the comment.
I want to do that in stages though, the current beta I would like to release first as stable release (waiting for some feedback)
Then I would like to continue to deprecate NC25 and to deal with the deprecation and to remove the code snippets to catch the old version.
And with the vue-rewrite we will see how fast that will go :)